tesseract_motion_planners/ompl/src/ompl_motion_planner.cpp

```diff
@@ -181,21 +181,9 @@ std::pair<bool, std::string> parallelPlan(ompl::geometric::SimpleSetup& simple_s
   if (solver_config.simplify)
     simple_setup.simplifySolution();
 
-  // Interpolate the path if it shouldn't be simplified and there are currently fewer states than requested
+  // Interpolate the path there are currently fewer states than requested
```

Comment:
Minor typo. Should be "Interpolate the path **if** there are..."

Reply:
Updated